### DISCUSSION TITLE
config_pgcluster.yml usage

### DISCUSSION THREAD

**Discussion by zhiqiangwang08** [autobase-tech/autobase#520] [General]
any explanation about config_pgcluster.yml? 
what is the usage scenario?
what's the difference with deploy_pgcluster.yml?

Thanks

**Comment by vitabaks** [comment]
deploy - performed once during the first deployment of the cluster

config - used for further maintenance of the cluster (changing parameters, adding databases, etc.)
Details here https://github.com/vitabaks/postgresql_cluster#using-git-for-cluster-configuration-management-iacgitops
